tests/cypress/e2e/components/media.cy.ts

```diff
@@ -98,7 +100,38 @@ describe('Media component', () => {
   mediaType.forEach(
     ({ url, title, description, collection, creator, tagRelated, type }) => {
       it(`should render ${type} in Media component`, () => {
+        const isRmrk = url.startsWith('/rmrk')
+
+        if (isRmrk) {
+          cy.intercept('POST', '**/rubick/007/graphql', (req) => {
```

Comment:
how about don't specify the version?
```suggestion
          cy.intercept('POST', '**/rubick/*/graphql', (req) => {
```

Reply:
very good thought. did the same for snek